### DISCUSSION TITLE
snmp not working ?

### DISCUSSION THREAD

**Discussion by AlexIT-FT** [accel-ppp/accel-ppp#153] [General]
Hi,
just make from source v1.12 under debian buster with
-DCPACK_TYPE=Debian10
-DBUILD_IPOE_DRIVER=TRUE
-DBUILD_VLAN_MON_DRIVER=TRUE
-DRADIUS=TRUE -DNETSNMP=TRUE 

tested with two config file

the first with
[snmp]
master=0
agent-name=accel-ppp

the second with
[snmp]
master=1

no errors (it seems)

but ... snmp not working
**snmpwalk -v 2c -c public 127.0.0.1 .1.3.6.1.4.1.8072.100**
just returns 
**iso.3.6.1.4.1.8072.100 = No Such Object available on this agent at this OID**

other OIDs  (ex .1.3.6.1.4.1.8072.1.5) work well

is it a bug, or am I missing something?

Thanks

**Comment by gerritavenant** [comment]
Hi @AlexIT-FT, did you figure out what your issue was?
I see something similar and in my accel-pppd debug logs I see the following:

```
[2025-09-22 18:15:15.662] net-snmp: Warning: no access control information configured.
  (Config search path: /etc/snmp:/usr/share/snmp:/usr/lib/x86_64-linux-gnu/snmp:/root/.snmp)
  It's unlikely this agent can serve any useful purpose in this state.
  Run "snmpconf -g basic_setup" to help you configure the accel-ppp.conf file for this agent.

[2025-09-22 18:15:15.662] net-snmp: Error opening specified endpoint ""
```
